domain/src/Aggregates/SharePoolingAsset/Actions/DisposeOfSharePoolingAsset.php

```diff
@@ -16,13 +17,14 @@ public function __construct(
         public LocalDate $date,
         public Quantity $quantity,
         public FiatAmount $proceeds,
-        public ?int $position = null,
+        // Only present whenever the disposal has been reverted and is now being replayed
+        public ?SharePoolingAssetTransactionId $id = null,
```

Comment:
ℹ️ This is also used for testing purposes. Again, not great, but I need to be able to identify the replayed disposal from the aggregate's list of transactions somehow, which is currently done through its ID.